test/cmdlineTests/storage_layout_specifier_repeated_definition_error/output.json

```diff
@@ -0,0 +1,37 @@
+{
+    "errors": [
+        {
+            "component": "general",
+            "errorCode": "8714",
+            "formattedMessage": "ParserError: More than one storage layout definition.
+ --> fileA:5:34:
+  |
+5 | contract C layout at 0x1234 is A layout at 0xABCD {
+  |                                  ^^^^^^
+Note: Previous definition:
+ --> fileA:5:12:
+  |
+5 | contract C layout at 0x1234 is A layout at 0xABCD {
+  |            ^^^^^^^^^^^^^^^^^^^
```

Comment:
The location for the layout specifier is not exactly correct. Note that it catches the `is`, which does not belong to it.

BTW, we should have an assert that the layout specifier location and the inheritance list location do not overlap.

Reply:
Looks like the location is correct now. Still needs the assert though.

Reply:
Fixed.
Regarding the assertion that layout specifier location does not overlap the inheritance list location, the latter does not include the location of the token `is`. The token is not stored in the AST Node during parsing.

Reply:
To be clear, I did added the assert for the locations, but it doesn't cover the `is` token as explained before.